samples/core/src/main/resources/db/migration/mysql/V0002__R001_Master_data_MYSQL.sql.mysql

```diff
@@ -50,6 +50,6 @@ INSERT INTO BILLORDERPOSITION (billId, orderPositionsId) VALUES (2,3);
 INSERT INTO BILLORDERPOSITION (billId, orderPositionsId) VALUES (2,4);
 
 INSERT INTO STAFFMEMBER (id, modificationCounter, role, login, firstname, lastname) VALUES (1, 0, 3, 'chief', 'Charly', 'Chief');
-INSERT INTO STAFFMEMBER (id, modificationCounter, role, login, firstname, lastname) VALUES (2, 0, 0, 'cook', 'Carl', 'Cook');
+INSERT INTO STAFFMEMBER (id, modificationCounter, role, login, firstname, lastname) VALUES (2, 0, 1, 'cook', 'Carl', 'Cook');
```

Comment:
@amarinso As with other databases, MariaDB does not support 0 as valid sequence value. For that reason, sequence for table STAFFMEMBER in the file above starts with 1 rather than 0. This change failed one of the tests, because the passed Waiter id '2' in case of MariaDB happens to be that of cook and not waiter. So in order for test case to work for MaridaDB I will revert this change and add note to the MariaDB documentation to make appropriate changes in the test to pass the correct water id.